src/main/java/software/amazon/awssdk/crt/CRT.java

```diff
@@ -29,7 +28,7 @@ public final class CRT {
     private static final String CRT_ARCH_OVERRIDE_SYSTEM_PROPERTY = "aws.crt.arch";
     private static final String CRT_ARCH_OVERRIDE_ENVIRONMENT_VARIABLE = "AWS_CRT_ARCH";
 
-    private static final String CRT_LIB_NAME = "aws-crt-jni";
+    public static final String CRT_LIB_NAME = "aws-crt-jni";
```

Comment:
you just showe me an example where you didn't move the shared-lib along with the native image, but the error message was about how it couldn't extract the shared-lib from the JAR. But that doesn't apply at all anymore to GraalVM users

Please get the error messages to fit the new behavior and point people in the right direction

Reply:
I don't think there is a decent way to tell graalvm apart from normal jvm here. :(

Reply:
actually `org.graalvm.nativeimage.imagecode` works